### ISSUE TITLE
Build fail on ROS2 crystal

### ISSUE THREAD

**Issue by revizes** [ApexAI/performance_test#43]
Fail log :
/usr/include/boost/mpl/for_each.hpp:118:46:   required from ‘void boost::mpl::for_each(F, Sequence*) [with Sequence = boost::mpl::list<performance_test::topics::Array1k, performance_test::topics::Array4k, performance_test::topics::Array16k, performance_test::topics::Array32k, performance_test::topics::Array60k, performance_test::topics::Array1m, performance_test::topics::Array2m, performance_test::topics::Struct16, performance_test::topics::Struct256, performance_test::topics::Struct4k, 
performance_test::topics::Struct32k, performance_test::topics::PointCloud512k, performance_test::topics::PointCloud1m, performance_test::topics::PointCloud2m, performance_test::topics::PointCloud4m, performance_test::topics::Range, performance_test::topics::NavSatFix, performance_test::topics::RadarDetection, performance_test::topics::RadarTrack>; F = performance_test::DataRunnerFactory::get(const string&, performance_test::CommunicationMean, performance_test::RunType)::<lambda(auto:9)>]’
/home/robot/ros2_ws/perf_test_ws/src/performance_test/performance_test/src/data_running/data_runner_factory.cpp:63:6:   required from here
/home/robot/ros2_ws/perf_test_ws/src/performance_test/performance_test/src/data_running/../communication_abstractions/fast_rtps_communicator.hpp:133:18: error: invalid new-expression of abstract class type ‘performance_test_msgs::msg::dds_::Array1k_PubSubType’
     m_topic_type(new TopicType())
                  ^~~~~~~~~~~~~~~

Problem is my environment? Can you give me some advice?

**Comment by rutvih** [comment]
@revizes This [PR](https://github.com/ApexAI/performance_test/pull/42) is submitted by my team to fix build on Crystal. Let me know if that helps you with the issue you're facing.

**Comment by deeplearningrobotics** [comment]
@revizes: We are incoperating @rutvih and other fixes for Crystal as we speak. I'll keep you updated.

**Comment by revizes** [comment]
@rutvih & @deeplearningrobotics : Sorry for late.
I removed all ros packages from my pc. And I tried to build full source code of ros2.
Then, re-sync performance_test git and build.

first time was failed. second time was successful. ( I don't know why first time was failed.)

anyway i executed below commands

$ os2 run performance_test perf_test -c ROS2 -l log -t Array1k --max_runtime 10
blar blar blar
Maximum runtime reached. Exiting.

$ ros2 run performance_test performance_test_file_reader.py .
Skipping .
Parsing file:./log_Array1k_27-02-2019_15-41-41
Could not parse file: ./log_Array1k_27-02-2019_15-41-41

Thanks.
